CHANGELOG.md

```diff
@@ -0,0 +1,174 @@
+# CHANGELOG
+
+This is a manually generated log to track changes to the repository for each release. 
+Each section should include general headers such as **Implemented enhancements** 
+and **Merged pull requests**. All closed issued and bug fixes should be 
+represented by the pull requests that fixed them. This log originated with Singularity 2.4
+and changes prior to that are (unfortunately) done retrospectively. From @olifre, critical items to know are:
+
+ - renamed commands
+ - deprecated / removed commands
+ - changed defaults
+ - backward incompatible changes (recipe file format? image file format?)
+ - migration guidance (how to convert images?)
+ - changed behaviour (recipe sections work differently)
+
+
+## [v3.0](https://github.com/singularityware/singularity/tree/development) (development)
```

Comment:
Should probably be 2.4.1 not 3.0.

Reply:
@gmkurtzer said the next "official" version is 3.0, but I suppose we didn't plan for the bug fixes. I'll change it if it makes you happy.

Reply:
It's either that or go without a changelog in 2.4.1.   😺